### ISSUE TITLE
Build slope.qml

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/RiverscapesXML#58]
Please create a new `slope.qml` equivalent to [slope.lyr](https://github.com/Riverscapes/RiverscapesXML/blob/master/Symbology/esri/Shared/slope.lyr)

**Comment by joewheaton** [comment]
This is really similar to #57

**Comment by MattReimer** [comment]
This looks great!

***FYI @shelbysawyer and @lauren-herbine and @joewheaton***: if you add `"fixes #58"` or `"closes #58"` to the end of your commit message then this issue will automatically be closed for you whenever the pull request goes through.

so something like "Adding Slope Symbology fixes #58"

**Comment by joewheaton** [comment]
@lauren-herbine and @shelbysawyer.... just for kicks on instant gratification, 
- 6 hours ago @philipbaileynar explained this foreign concept to us of symbology updates for qRAVE
- 1 hour ago @shelbysawyer added this pull request. 13 minutes ago @MattReimer 
- 13 minutes ago @MattReimer approved the pull request and Merged this...
- Now I tried to test it on some other projects and voilà 
[![](http://img.youtube.com/vi/Genem0SCdV8/0.jpg)](http://www.youtube.com/watch?v=Genem0SCdV8 "")

In other words... 45 minute turn around with @MattReimer approving these things and your changes are now benefitting all of us with RS Context, VBET, and BRAT projects and wanting to work in QRAVE and Q (not ArcGIS... yes you @nick4rivers)!

**Comment by nick4rivers** [comment]
This is awesome!! Joint the Q!!

**Comment by lauren-herbine** [comment]
Thanks for the morale boost @joewheaton !

**Comment by shelbysawyer** [comment]
Awesome!! Thanks for the lesson @philipbaileynar and the support @joewheaton!
